### PR TITLE
Refactoring/39 mutli module ci build

### LINKED ISSUE #39: Mutli-module CI build
## Situation

Currently the CI build scripts created by Project Keeper are for a single module Maven build. Since there are multiple tutorials in this project, we use a multi-module Maven build. As a result the standard CI builds don't look for the build artifacts (JARs) in the right directory. They look in 'target/', but there are multiple directories in the form of `<sub-module>/target/` that actually need to be scanned. 

Additionally, we need to take the release artifacts from those directories.

## Acceptance Criteria

* CI handles multi-module build correctly

### PR DESCRIPTION
**Pull request by redcatbear**
Closes #39.

### CODE REVIEW

File: .github/workflows/release_droid_prepare_original_checksum.yml
Comment (kaklakariada) [inline .github/workflows/release_droid_prepare_original_checksum.yml:1]: This is edited manually, please remove it from .gitattributes.

File: .github/workflows/release_droid_print_quick_checksum.yml
Comment (kaklakariada) [inline .github/workflows/release_droid_print_quick_checksum.yml:1]: This is edited manually, please remove it from .gitattributes.

File: .github/workflows/release_droid_prepare_original_checksum.yml [hunk 1]
```diff
@@ -22,7 +22,7 @@ jobs:
       - name: Run tests and build with Maven
         run: mvn --batch-mode clean verify --file pom.xml
       - name: Prepare checksum
-        run: find target -maxdepth 1 -name *.jar -exec sha256sum "{}" + > original_checksum
+        run: find . -regex '.*/target/.*\.jar' -exec sha256sum "{}" + > original_checksum
```
Comment (kaklakariada) [inline .github/workflows/release_droid_prepare_original_checksum.yml:25]: Does this also work for single-module projects? If yes we should use the same mechanism in project-keeper.

**Review by kaklakariada — CHANGES_REQUESTED** [review]

File: .github/workflows/release_droid_prepare_original_checksum.yml (hunk 1 above)
Comment (redcatbear) [reply .github/workflows/release_droid_prepare_original_checksum.yml:25]: My short experiment looked good. It found `./target/<blah>.jar`

**Review by kaklakariada — APPROVED** [review]

File: .github/workflows/release_droid_prepare_original_checksum.yml (hunk 1 above)
Comment (kaklakariada) [reply .github/workflows/release_droid_prepare_original_checksum.yml:25]: Created https://github.com/exasol/project-keeper/issues/474 to add this to PK

**Pull request merged**